src/vs/workbench/services/lifecycle/electron-sandbox/lifecycleService.ts

```diff
@@ -81,19 +74,6 @@ export class NativeLifecycleService extends AbstractLifecycleService {
 		});
 	}
 
-	private async handleRemoteAgentDisconnect(): Promise<void> {
```

Comment:
The removal here was the incorrectly merged https://github.com/microsoft/vscode/pull/219041, containing code that was drafted for but later cut from https://github.com/microsoft/vscode/pull/218972